packages/documentation/src/content/docs/concepts/interledger-protocol/peering.md

```diff
@@ -323,3 +324,60 @@ Example successful response:
   }
 }
 ```
+
+## Auto-peering
+
+Additionally, certain peers (testnet?) will have _auto-peering_ available. This feature is not (and should not) be used in production enviornments, however, it allows local and staging environment Rafiki instances to environments for easier peering integration. This requires the peer you want to peer with to publish an "auto-peering" url. Once this `peerUrl` is provided, and it is communicated that the peer supports auto-peering, instead of using `createPeer` mutation to create a peer, you can call `createPeerByUrl`:
```

Comment:
nit: clean up `(testnet?)`? Or update the "certain peers" phrasing to include it